network/config.go

```diff
@@ -79,6 +79,9 @@ type Config struct {
 	Name string `json:"name"`
 	// Backend specifies the backend for the network
 	Backend Backend `json:"backend"`
+	// Flags can hold additional flags for avalanchego nodes.
+	// It can be empty.
+	Flags map[string]interface{}`json:"flags"`
```

Comment:
Can you document here that the precedence from highest to lowest is:
1, Config file
2. Flags defined in network.Config
3. Flags defined in node.Config